etc/helm/examples/local-dev-values.yaml

```diff
@@ -50,7 +50,7 @@ pachd:
 console:
   enabled: false
   config:
-    issuerURI: "http://pachd:1658/"
+    issuerURI: "http://pachd:1658"
```

Comment:
is this trailing slash removed on purpose? I think `console.issuerURI `must equal `pachd.oauthIssuer`

Reply:
based on @cpepin 's  comments, I would think we should remove the trailing slash from `pachd.oauthIssuer`